docs/tutorials/making-games-jettycat.md

```diff
@@ -917,4 +915,4 @@ Try changing this stuff to train yourself in coding:
 * Change the cat's movement so that it is more close to what happens in Flappy Bird: make the cat fly upwards abruptly when a player taps the screen, but do nothing if they then press the screen continuously.
 * Make rotating tubes to make the game more challenging.
 * Add a life counter, and allow a player to take 3 hits before losing.
-* Add sounds! Visit ct.sound documentation inside your editor's docs for modules on how to play sounds in your game.
+* Add sounds! Visit sound documentation inside your editor's docs for modules on how to play sounds in your game.
```

Comment:
The sound API is now a part of core library, so there should be a link to [this page](https://deploy-preview-128--clever-shaw-0cdd72.netlify.app/sounds.html) (probably `../sounds.md`) instead of referring to catmods